nncf/quantization/algorithms/weight_compression/mixed_precision.py

```diff
@@ -360,7 +360,7 @@ def _calc_weight_sensitivity(
         return fns.linalg.norm(decompressed_weight - weight, ord="fro").item()
 
     def _get_statistic_collector(self):
-        return self._backend_entity.hawq_statistic_collector()
+        return self._backend_entity.hawq_statistic_collector(self._subset_size)
```

Comment:
While we are here. Could you please add back a `self` argument to the definition of the base `_get_statistic_collector` method? I've missed this during the last review.
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/quantization/algorithms/weight_compression/mixed_precision.py#L284

Reply:
sure